app/page.tsx

```diff
@@ -17,7 +17,7 @@ export default function MainPage() {
   return (
     <div style={containerStyling}>
       <Box>
-        <Image src="/logo.svg" width={450} height={100} alt="Neighbourhood Logo" />
+        <Image src="/logo.svg" width={450} height={100} alt="Neighbourhood Logo" priority />
```

Comment:
what is this priority property

Reply:
tbh idk it gives a warning in dev tools when ur on the homepage so I just slapped it on 🤷‍♀️
for our main logo, supposed to be used when you have a large resource

Reply:
LOL fair enough makes sense